ql/src/java/org/apache/hadoop/hive/ql/exec/MoveTask.java

```diff
@@ -317,6 +320,36 @@ public int execute() {
     }
 
     try (LocalTableLock lock = acquireLockForFileMove(work.getLoadTableWork())) {
+      String storageHandlerClass = null;
```

Comment:
Would it make sense to move this newly-added code into a method? e.g. `boolean checkAndCommitNatively(work)` or something like that, with some javadoc. What do you think?

Reply:
Refactored out code